config.test.yaml

```diff
@@ -39,6 +39,7 @@ templates:
       - mediawiki/v1/content
       - mediawiki_v1_graphoid
       - mediawiki/v1/mobileapps
+      - analytics_v1_pageviews
```

Comment:
We should think carefully on the naming scheme, because we will have 2 analytics configs, and we really don't want to mess them up.

I propose the following:
1. `/analytics/v1/pageviews.yaml` - the whole analytics folder would belong to backend analytics RESTBase.
2. `/data/v1/analytics.yaml` - here we would put all publicly-exposed endpoints in the front-end RESTBase.

@gwicke @d00rman @eevans Any thoughts on the naming scheme?